cmd/bankapp/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/iliyanmotovski/bankv3/bank/api"
+	"github.com/iliyanmotovski/bankv3/bank/domain"
+	"github.com/iliyanmotovski/bankv3/bank/persistence"
+	"github.com/justinas/alice"
+	"gopkg.in/mgo.v2"
+	"net/http"
+	"time"
+)
+
+func main() {
+	session, err := mgo.DialWithTimeout("localhost:27016", time.Second*10)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	defer session.Close()
+	session.SetMode(mgo.Monotonic, true)
+
+	r := mux.NewRouter()
+	s := r.PathPrefix("/v3/users").Subrouter()
+
+	userSessionDuration := time.Second * 300
+	mongoSessionStore := persistence.NewMongoSessionStore(*session, "bank")
+
+	userStore := domain.UserStore(mongoSessionStore)
+	userSessionStore := domain.SessionStore(mongoSessionStore)
+	accountStore := domain.AccountStore(mongoSessionStore)
+
+	SignUpHandlers := alice.New(api.LoggingMiddleware, api.RecoverMiddleware)
+	SecurityHandlers := alice.New(api.LoggingMiddleware, api.RecoverMiddleware, api.CookieBasedSecurity(userSessionStore, userSessionDuration))
+
+	r.Handle("/", http.FileServer(http.Dir("./cmd/bankapp/static")))
```

Comment:
should not pass relative path. "static" should be enough as it's already in cmd/bankapp